docker-compose.yml

```diff
@@ -178,6 +178,13 @@ services:
     depends_on:
       - external_redis
 
+  elasticsearch:
+    build:
+      context: ./elasticsearch/2.4/
```

Comment:
the other Dockerfile-*'s are only committed as CP doesn't have a pre-build step that can run build.sh